deploy.sh

```diff
@@ -1,13 +1,27 @@
 #!/bin/bash
 
+# Deploying means to package and make available a Helm chart in the Helm chart
+# repostiory hosted as a GitHub pages website. GitHub pages websites are hosted
+# by GitHub automatically if we push to a github repository of a certain branch
+# called gh-pages.
+#
+# See:
+# - gh-pages branch: https://github.com/pangeo-data/helm-chart/tree/gh-pages
+# - Rendered GitHub pages: https://pangeo-data.github.io/helm-chart/
+
+# We need to provide credentails to push to the the git repository configured in
+# chartpress.yaml, which is this git repo. The push will be made by chartpress
+# and be made to the gh-pages branch specifically.
 set -eu
 openssl aes-256-cbc -K $encrypted_058f1ad78f7f_key -iv $encrypted_058f1ad78f7f_iv -in deploy-key.rsa.enc -out deploy-key.rsa -d
 set -x
 chmod 0400 deploy-key.rsa
-helm init --client-only
-helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
-helm repo update
-helm dependency update pangeo
```

Comment:
I'm quite confident that adding a named repository isn't needed unless we reference the named repository somewhere, which for anything, and we don't.

Also, currently chartpress will update dependencies in requirements.yaml before packaging a chart using helm.

Reply:
Agreed. I think the fact that [the JupyterHub repo is referenced explicitly in the requirements](https://github.com/pangeo-data/helm-chart/blob/master/pangeo/requirements.yaml#L5) means this isn't necessary.